src/Rector/Deprecation/StaticArgumentRenameBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation;
+
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+use Rector\Core\RectorDefinition\CodeSample;
+use Rector\Core\RectorDefinition\RectorDefinition;
+
+/**
+ * Renames deprecated the argument in a static call.
+ */
+abstract class StaticArgumentRenameBase extends AbstractRector {
+
+  /**
+   * The fully qualified class name.
+   *
+   * @var string
+   */
+  protected $fullyQualifiedClassName;
+
+  /**
+   * The method name.
+   *
+   * @var string
+   */
+  protected $methodName;
+
+  /**
+   * The deprecated argument.
+   *
+   * @var string.
+   */
+  protected $deprecatedArgument;
+
+  /**
+   * The replacement argument.
+   *
+   * @var string.
+   */
+  protected $argument;
+
+  /**
+   * @inheritdoc
+   */
+  public function getNodeTypes(): array {
+    return [
+      Node\Expr\StaticCall::class
+    ];
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public function refactor(Node $node): ?Node {
+
+    if ($node instanceof Node\Expr\StaticCall) {
+      /** @var Node\Expr\StaticCall $node */
+      if ($this->getName($node->name) === $this->methodName && (string) $node->class === $this->fullyQualifiedClassName) {
```

Comment:
`$this->getName($node->name) === $this->methodName` is the way we've started checking the names of nodes. It seems a little more straight forward than the other ways we've used.